lms/templates/instructor/instructor_dashboard_2/student_admin.html

```diff
@@ -113,6 +120,51 @@
   </div>
 %endif
 
+%if settings.FEATURES.get('ENABLE_INSTRUCTOR_BACKGROUND_TASKS') and section_data['access']['instructor']:
+    <hr>
+    <div class="course-specific-container action-type-container">
+        <h2>${_('Submit External Grades')}</h2>
+        <div class="request-response-error"></div>
+        <p>
+            ${_("To import grades from an assignment outside of the Lagunita platform, "
+            "ask the Course Operations team to set up external grading for your course. "
+            "Then download the grade submission CSV with the button below and fill in the missing fields. "
+            "Once you have saved your changes, select an assignment you are importing from the drop-down. "
+            "Finally, upload and submit your grades" )}
+        </p>
+        <p><input type="button" name="lti-blank-grades" value="${_("Download Blank Grade Submission CSV")}" data-csv="true" class="csv" data-endpoint="${ section_data['get_blank_lti_url'] }"></p>
+
+        <form id="lti-upload-form" method="post" action="${ section_data['upload_lti_url'] }" enctype="multipart/form-data">
+            <h4 class="new-section">${_('Select Assignment')}</h4>
+            <select class="endpoint-selector select-tag new-section" name="lti-endpoint"></select>
+            <div class="form-actions new-section">
+                <label for="lti-key">LTI Key</label>
+                <input type="text" class="textfield-tag" name="lti-key">
+            </div>
+            <div class="form-actions">
```

Comment:
Also, are the key and secret things instructors are expected to know? the UI mockup did not have these fields